### DISCUSSION TITLE
Is there a way to include a module in pyodine ?

### DISCUSSION THREAD

**Discussion by Starmania** [pyodide/pyodide#5083] [Q&A] [answered]
Hey, I would like to know if this is possible to include a module directly in pyodine? I don't plan to use public cdn (I will serve every files by myself) so I thought that I can directly include the modules I need in the build but is this a supported behavior? 
Is there another solution?

**Comment by ryanking13** [comment, marked as answer]
I think you can try this: https://github.com/pyodide/pyodide/discussions/4575
